.pre-commit-config.yaml

```diff
@@ -5,6 +5,22 @@ ci:
   autoupdate_commit_msg: ":arrow-up: dep-bump(pre-commit): update pre-commit hooks"
 
 repos:
+  - repo: local
+    hooks:
+      - id: ruff-format
+        name: ruff-format
+        entry: "bin/ruff-format"
+        language: script
+        pass_filenames: true
+        files: "src/python/|test/python/"
+        types: [ python ]
+      - id: ruff-check
+        name: ruff-check
+        entry: "bin/ruff-check"
+        language: script
+        pass_filenames: true
+        files: "src/python/|test/python/"
+        types: [ python ]
```

Comment:
can you add `require_serial: true` in these two hooks?
This is how ruff does it: https://github.com/astral-sh/ruff-pre-commit/blob/596470fba20d04adc68ec7903ff69a12e5e1a8d3/.pre-commit-hooks.yaml#L8

Reply:
will do tonight!